### PR TITLE
[JAVA] API 2.0 face detection demo

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

**Comment by likholat** [comment]
@allnes

File: modules/java_api/samples/java_api_2.0_sample/Main.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import org.intel.openvino.*;
+import org.intel.openvino.Core;
+import org.opencv.core.*;
+import org.opencv.highgui.HighGui;
+import org.opencv.imgcodecs.*;
+import org.opencv.imgproc.Imgproc;
```
Comment (dkurt) [inline modules/java_api/samples/java_api_2.0_sample/Main.java:6]: I'd like recommend replace deprecated sample completely. It's OK to keep both APIs in the tests but samples should propose the latest one.

File: modules/java_api/samples/java_api_2.0_sample/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2020 Intel Corporation
```
Comment (dkurt) [inline modules/java_api/samples/java_api_2.0_sample/CMakeLists.txt:1]:
```
# Copyright (C) 2020-2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

File: modules/java_api/samples/java_api_2.0_sample/Main.java [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import org.intel.openvino.*;
```
Comment (dkurt) [inline modules/java_api/samples/java_api_2.0_sample/Main.java:1]:
```
# Copyright (C) 2020-2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

File: modules/java_api/samples/java_api_2.0_sample/Main.java (hunk 1 above)
Comment (likholat) [reply modules/java_api/samples/java_api_2.0_sample/Main.java:6]: I agree, but I think we should first add async methods for Java API 2.0.

Comment (dkurt) [reply modules/java_api/samples/java_api_2.0_sample/Main.java:9]: These are two different demos so why not upgrade a synchronous sample first?

**Review by dkurt — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**